### PR TITLE
Improve ug

### PR DESCRIPTION
**Pull request by richiehx**

### CODE REVIEW

**Review by yiwen101 — COMMENTED** [review]
Emm, how about change SCHEDULE, STUDENT and TASK to lower case to standardize? I thought Jung says we  stick to either bold or italic, and do not do upper case.

File: docs/team/richiehx.md [hunk 1]
```diff
@@ -5,42 +5,46 @@ title: "Richie's Project Portfolio Page"
...
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to show the details of the selected student/lesson/task.
-  * What it does: allows the user to display the selected item in the show details panel at the right of the User Interface. It allows the user to show students while in the `STUDENT` list, show lessons while in the `SCHEDULE` list and show tasks while in the `TASK` list.
+* **New Feature**: Added the ability to show the details of the selected student/lesson/task. (Pull request [#53](https://github.com/AY2324S1-CS2103T-T11-3/tp/pull/53), [#138](https://github.com/AY2324S1-CS2103T-T11-3/tp/pull/138))
+  * What it does: allows the user to display the selected item in the show details panel at the right of the User Interface. It allows the user to show students while in the ___STUDENT list___, show lessons while in the ___SCHEDULE list___ and show tasks while in the ___TASK list___.
```
Comment (yiwen101) [inline docs/team/richiehx.md:13]: Emm, how about change SCHEDULE, STUDENT and TASK to lower case to standardize? I thought Jung says we stick to either bold or italic, and do not do upper case.

Comment (yiwen101) [reply docs/team/richiehx.md:13]: I approve first

**Review by yiwen101 — APPROVED** [review]

File: docs/team/richiehx.md (hunk 1 above)
Comment (richiehx) [reply docs/team/richiehx.md:13]: In my opinion, everything else just doesn't look good. You guys can try it out and I will standardise to you all.

**Review by pzl111 — APPROVED** [review]

**Pull request merged**